how-to-use-azureml/automated-machine-learning/forecasting-bike-share/auto-ml-forecasting-bike-share.yml

```diff
@@ -0,0 +1,9 @@
+name: auto-ml-forecasting-bike-share
```

Comment:
This file is not in AzureMlCli and should not be pushed to GitHub.
It is generated as part of running the associated notebook.